e2e/tests/uploadDraftOrders_test.js

```diff
@@ -79,8 +81,8 @@ Scenario('Local authority uploads draft orders', async ({I, caseViewPage, upload
   I.seeInTab(['Further evidence documents for hearings 1', 'Documents 1', 'Document name'], supportingDoc.name);
   I.seeInTab(['Further evidence documents for hearings 1', 'Documents 1', 'Notes'], supportingDoc.notes);
   I.seeInTab(['Further evidence documents for hearings 1', 'Documents 1', 'File'], supportingDoc.fileName);
-  I.seeInTab(['Further evidence documents for hearings 1', 'Date and time uploaded', dateFormat(today, 'd mmm yyyy')]);
-  I.seeInTab(['Further evidence documents for hearings 1', 'Uploaded by', 'kurt@swansea.gov.uk']);
+  I.seeInTab(['Further evidence documents for hearings 1', 'Date and time uploaded'], date);
```

Comment:
very odd that you need to make this change?

Reply:
yeah not sure why this was needed to be honest!

Reply:
but it is probs a good one so that you aren't redeclaring the same thing multiple times